src/tree/subscriptionTreeItem.ts

```diff
@@ -27,23 +28,23 @@ export default class SubscriptionTreeItem extends SubscriptionTreeItemBase imple
     }
 
     public async loadMoreChildrenImpl(clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
-        const client = new ResourceManagementClient(this.root.credentials, this.root.subscriptionId);
+        const client = new ResourceManagementClient(this.subscription.credentials!, this.subscription.subscriptionId!);
```

Comment:
I wonder why credentials can now be nullable/undefined? Can we be certain it's populated at this point? Do the docs give any guidance about under what circumstances it might not be set, and what to do about it if that's the case?

Just wondering if there's some re-authenticating dance we may need to do.

Reply:
Nice, I think I was playing around kept `!` by mistake, changed, and just tested it as well. Thank you for noticing this.